dev-haskell/hlint/hlint-3.2.3.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.6.9999
+
+COMMIT="6c6593e0ec0fe45640879946a181f706c2a38972"
+CABAL_FEATURES="lib profile haddock hoogle hscolour"
+inherit haskell-cabal
```

Comment:
Optional, but leaving a note here as you requested in IRC: inherit `elisp-common` after `haskell-cabal`. Add in an `emacs` IUSE. And further down, add in the ebuild phases from `hlint-2.1.17-r1` that do emacs lisp stuff.

To see what's needed to migrate over, run `diff -u --color hlint-2.1.17-r1.ebuild hlint-3.2.3.ebuild`.

Reply:
This is done now, I've confirmed that hlint builds and installs properly with the changes, though I can't speak for the emacs stuff itself as I do not know how to use emacs.